utils/torch_solver.py

```diff
@@ -57,11 +57,12 @@ def set_objective(self, model_init_fn, dataset):
 
         # TODO: num_worker should not be hard coded. Finding a sensible way to
         # set this value is necessary here.
-        system = os.environ.get('RUNNER_OS', platform.system())
-        is_mac = system == 'Darwin' or system == 'macOS'
+        system = os.environ.get('RUNNER_OS', sys.platform)
```

Comment:
I saw in an SO answer (will find link), that `platform` gives a more reliable system than `sys.platform`. Did you change for a particular reason ?

Reply:
No real reason I guess, mostly out of habit.
I found the [SO answer](https://stackoverflow.com/questions/1854/python-what-os-am-i-running-on)

To me, `sys.platform` avoid any dark magic as for `osx` it always returns `darwin` so as it is used under the hood, I would directly use this but I am fine with reverting to `platform`